ui/README.md

```diff
@@ -15,7 +15,7 @@ For purposes of actively developing Meshery UI, you first need to ensure you hav
     > **NOTE:** Please run the steps in order to avoid issues, as Meshery server should be running and logged-in before accessing the development server
     > on 3000 port.
 
-    > **NOTE:** Its strongly recommended to use  either [Node Version Manager](https://github.com/nvm-sh/nvm#node-version-manager---)) in linux/mac os systems or [NVM for Windows](https://github.com/coreybutler/nvm-windows#nvm-for-windows) on Windows systems so single `nvm use` / `nvm install` simplifies installing and using correct node version locallly **(v14.8.0)**, see [NVM Intro](https://github.com/nvm-sh/nvm#intro) for details. Otherwise, you might experience issues during local `npm i` similar to [4674](https://github.com/meshery/meshery/issues/4674) due to how optional dependencies are resolved in npm v6.
+    > **NOTE:** Its strongly recommended to use  either [Node Version Manager](https://github.com/nvm-sh/nvm#node-version-manager---)) in linux/mac os systems or [NVM for Windows](https://github.com/coreybutler/nvm-windows#nvm-for-windows) on Windows systems so single `nvm use` / `nvm install` simplifies installing and using correct node version locallly **(v18.13.0)**, see [NVM Intro](https://github.com/nvm-sh/nvm#intro) for details. Otherwise, you might experience issues during local `npm i` similar to [4674](https://github.com/meshery/meshery/issues/4674) due to how optional dependencies are resolved in npm v6.
```

Comment:
Thank you for working on this issue @duckling69 . I think mentioning the major version is better since any minor version would work. Please correct me if i am wrong @Abhishek-kumar09 

So if you could mention v18, that would be better.

Reply:
Yeah